runconfig/opts/parse.go

```diff
@@ -337,6 +339,9 @@ func Parse(flags *pflag.FlagSet, copts *ContainerOptions) (*container.Config, *c
 		}
 	}
 
+	mounts := copts.mounts.Value()
+	// TODO: migrate binds and tmpfs to new API, and appends to mounts.
```

Comment:
I meant we should not set `HostConfig.Binds` 💦 , I'll update the comment for clarity